src/collective/cover/tiles/templates/carousel.pt

```diff
@@ -6,7 +6,7 @@
 
 <head>
   <link rel="stylesheet" type="text/css"
-      tal:attributes="href string:${context/@@plone_portal_state/portal_url}/++resource++collective.cover/css/galleria.cover_theme.css" />
+      tal:attributes="href string:${context/@@plone_portal_state/portal_url}/++resource++collective.cover/galleria/theme.css" />
```

Comment:
This may give cache problem.

Reply:
yes, but this was not changed here; we can fix it on another pull request.